### PR TITLE
APPS-6120 Updated module dependencies with new versions

### PR DESCRIPTION
**Pull request by oleksander-kiiashko**
Task: https://spryker.atlassian.net/browse/APPS-6120

### CODE REVIEW

File: docs/pbc/all/payment-service-providers/payone/payone.md [hunk 1]
```diff
@@ -79,10 +79,13 @@ When customers pay with PayPal, a shop owner can do the following:
...
+* `spryker/payment: ^5.10.0`
 * `spryker/sales: ^11.32.0`
 * `spryker/sales-return: ^1.4.0`
-* `spryker-shop/checkout-page: ^3.20.0`
-* `spryker-shop/payment-page: ^1.0.0`
+* `spryker-shop/checkout-page: ^3.21.0`
+* `spryker-shop/payment-page: ^1.2.0`
+* `spryker/oms: ^11.21.0`
+* `spryker/sales-payment: ^1.2.0`
+* `spryker/sales-oms: ^0.1.1`
```
Comment (matweew) [inline docs/pbc/all/payment-service-providers/payone/payone.md:91]: why the module spryker/sales-oms is required?

Comment (oleksander-kiiashko) [reply docs/pbc/all/payment-service-providers/payone/payone.md:91]: it was added with Order State Machine improvements
https://github.com/spryker/spryker/pull/9462

PR - https://github.com/spryker/spryker-docs/pull/1566/files

File: docs/pbc/all/ratings-reviews/202204.0/third-party-integrations/integrate-bazaarvoice.md [hunk 2]
```diff
@@ -8,15 +8,16 @@ template: howto-guide-template
...
-* `spryker/message-broker-extension: ^1.0.0`
+* `spryker/message-broker: ^1.3.0`
+* `spryker/message-broker-aws: ^1.3.2`
+* `spryker/message-broker-extension: ^1.1.0`
 * `spryker-shop/asset-widget: ^1.0.0`
+* `spryker-shop/cart-page: ^3.32.0`
 * `spryker-shop/product-detail-page: ^3.17.0`
 * `spryker-shop/product-category-widget: ^1.6.0`
-* `spryker-shop/shop-ui: ^1.59.0`
+* `spryker-shop/shop-ui: ^1.62.0`
```
Comment (danielsantos-spryker) [inline docs/pbc/all/ratings-reviews/202204.0/third-party-integrations/integrate-bazaarvoice.md:21]: There is new [release](https://release.spryker.com/release/pull-request/2329?type=3) for BazaarVoice in release queue right now so I suggest you add these:
- `spryker-shop/checkout-page: ^3.23.0`
- `spryker-shop/merchant-page: ^1.1.0`
- `spryker-shop/merchant-profile-widget: ^1.1.0`
- `spryker-shop/merchant-widget: ^1.3.0`
- `spryker-shop/payment-page: ^1.3.0`

Comment (oleksander-kiiashko) [reply docs/pbc/all/ratings-reviews/202204.0/third-party-integrations/integrate-bazaarvoice.md:21]: fixed here https://github.com/spryker/spryker-docs/pull/1566/files

**Pull request closed without merging**